subworkflows/local/assembly_coverage.nf

```diff
@@ -19,21 +19,21 @@ workflow ASSEMBLY_COVERAGE {
 
     ch_versions = ch_versions.mix(BWAMEM2_INDEX.out.versions)
 
-    BWAMEM2_MEM(
+    BWAMEM2_MEM_COVERAGE(
         reads,
         BWAMEM2_INDEX.out.index
     )
-   
-    ch_versions = ch_versions.mix(BWAMEM2_MEM.out.versions)
+
+    ch_versions = ch_versions.mix(BWAMEM2_MEM_COVERAGE.out.versions)
 
     METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS(
-        BWAMEM2_MEM.out.bam
+        BWAMEM2_MEM_COVERAGE.out.bam
     )
 
     ch_versions = ch_versions.mix(METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS.out.versions)
 
     SAMTOOLS_IDXSTATS(
-        BWAMEM2_MEM.out.bam
+        BWAMEM2_MEM_COVERAGE.out.bam
     )
 
```

Comment:
After stats calc, we should add a step to remove the bam files to minimise the time of storage:
`rm *.bam *.bai`

Reply:
Yes, we will migrate this to the GGP decontamination module (that one is much more storaga efficient)

Reply:
Also, the bam files here will be deleted once the pipeline completed successfully